test/integration/services.js

```diff
@@ -23,12 +23,12 @@ describe('app', () => {
           expect($('.services').text()).to.equal('Services at this surgery');
           expect($('.services').next('h3').text()).to.equal('Services patients can book themselves');
           expect($('.self-referrals li').length).to.equal(6);
-          expect($($('.self-referrals li')[0]).text()).to.equal('Asthma clinic');
```

Comment:
No. The extra service mapping is done in the ETL, not in Profiles.

The tests use the 'profiles-db-elastic' image, which is a few months old and doesn't contain the changed names.

When deployed profiles uses the Elasticsearch cluster which is updated every night, and contains the correct service names.

The integration tests are asserting that data held in the image is being rendered, and is less concerned with what the data is.

We have a situation where the elastic search image is only used in development, and will become increasing out of date as time goes on.